frontend/packages/local-storage-operator-plugin/src/components/disks-list/disks-list-page.tsx

```diff
@@ -172,26 +198,42 @@ export const NodesDisksListPage: React.FC<NodesDisksListPageProps> = ({
     },
   ];
 
+  const ErrorMessage = ({ message }) => {
+    return (
+      <Alert
+        isInline
+        className="co-alert co-alert--scrollable"
+        variant="danger"
+        title={t('public~An error occurred')}
+      >
+        <div className="co-pre-line">{message}</div>
+      </Alert>
+    );
+  };
+
```

Comment:
There is a prop in Table component `loadError`, have you tried that ?

Reply:
Looking into this.